woocommerce/woocommerce-android
# [Google for Woo] Dashboard Card Functionality part II (Stats)

closes: https://github.com/woocommerce/woocommerce-android/issues/11963
requires: https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/3052

please do not merge until `fluxCVersion` is updated to `trunk-[something]` and PR target is `trunk`


### Description


This PR makes it so that the Impressions and Clicks on the "Has Campaign" state of Google Ads dashboard card show the real number.


### Testing instruction

1. Pre-requisite: Install, activate, and set up the Google Ads plugin to a test site following pe5sF9-2Qo-p2. Ensure that you're using the plugin version 2.7.5 or above, and that there are existing campaigns already made. You can also ping me in Slack if you need invite to a test site,
2. Run app,
3. In My Store area, enable "Google Ads Campaigns" card in the Customize area, if it's not enabled yet,
4. Back in My Store, ensure "Google Ads Campaign" card is shown as loading and then shows the content for "Has Campaigns" screen.
5. Ensure that the numbers are showing correctly (should be zero for both if it's just a test site with no real campaigns)
